pkg/install/1-installresources.go

```diff
@@ -88,557 +82,569 @@ func (i *Installer) installResources(ctx context.Context) error {
+									},
+									Protocol:             mgmtnetwork.LoadBalancerOutboundRuleProtocolAll,
+									IdleTimeoutInMinutes: to.Int32Ptr(30),
 								},
+								Name: to.StringPtr("outboundrule"),
 							},
 						},
-						Name:     to.StringPtr("aro"),
-						Type:     to.StringPtr("Microsoft.Network/loadBalancers"),
-						Location: &installConfig.Config.Azure.Region,
-					},
-					APIVersion: apiVersions["network"],
-					DependsOn: []string{
-						"Microsoft.Network/publicIPAddresses/aro-outbound-pip",
 					},
+					Name:     to.StringPtr("aro"),
+					Type:     to.StringPtr("Microsoft.Network/loadBalancers"),
+					Location: &installConfig.Config.Azure.Region,
+				},
+				APIVersion: apiVersions["network"],
+				DependsOn: []string{
+					"Microsoft.Network/publicIPAddresses/aro-outbound-pip",
 				},
 			},
-		}
+		},
+	}
 
-		i.log.Print("deploying resources template")
-		err = i.deployments.CreateOrUpdateAndWait(ctx, resourceGroup, "azuredeploy", mgmtresources.Deployment{
-			Properties: &mgmtresources.DeploymentProperties{
-				Template: t,
-				Parameters: map[string]interface{}{
-					"sas": map[string]interface{}{
-						"value": map[string]interface{}{
-							"signedStart":         i.doc.OpenShiftCluster.Properties.Install.Now.Format(time.RFC3339),
-							"signedExpiry":        i.doc.OpenShiftCluster.Properties.Install.Now.Add(24 * time.Hour).Format(time.RFC3339),
-							"signedPermission":    "rl",
-							"signedResourceTypes": "o",
-							"signedServices":      "b",
-							"signedProtocol":      "https",
-						},
+	i.log.Print("deploying resources template")
+	err = i.deployments.CreateOrUpdateAndWait(ctx, resourceGroup, "azuredeploy", mgmtresources.Deployment{
+		Properties: &mgmtresources.DeploymentProperties{
+			Template: t,
+			Parameters: map[string]interface{}{
+				"sas": map[string]interface{}{
+					"value": map[string]interface{}{
+						"signedStart":         i.doc.OpenShiftCluster.Properties.Install.Now.Format(time.RFC3339),
+						"signedExpiry":        i.doc.OpenShiftCluster.Properties.Install.Now.Add(24 * time.Hour).Format(time.RFC3339),
+						"signedPermission":    "rl",
+						"signedResourceTypes": "o",
+						"signedServices":      "b",
+						"signedProtocol":      "https",
 					},
 				},
-				Mode: mgmtresources.Incremental,
 			},
-		})
-		if err != nil {
-			if isDeploymentActiveError(err) {
-				i.log.Print("waiting for resource template")
-				err = i.deployments.Wait(ctx, resourceGroup, "azuredeploy")
-			}
-			isQuota, errMsg := isResourceQuotaExceededError(err)
-			if isQuota {
-				return api.NewCloudError(http.StatusBadRequest, api.CloudErrorCodeQuotaExceeded, errMsg, "")
-			}
-			return err
+			Mode: mgmtresources.Incremental,
+		},
+	})
+	if err != nil {
+		if isDeploymentActiveError(err) {
+			i.log.Print("waiting for resource template")
+			err = i.deployments.Wait(ctx, resourceGroup, "azuredeploy")
+		}
+		isQuota, errMsg := isResourceQuotaExceededError(err)
+		if isQuota {
```

Comment:
I think there's a logic bug here - not yours, but please fix as a drive by.  This should surely be:
`} else if isQuota, errMsg := isResourceQuotaExceededError(err); isQuota {`